Library/test-helpers/lib.sh

```diff
@@ -1253,6 +1253,78 @@ limeCreateTestDir() {
 }
 
 
+true <<'=cut'
+=pod
+
+=head2 limeCopySourceFile
```

Comment:
Let's call it `limeCopyKeylimeFile`, and it should accept similar options as the other function. In addition, `-d|--download` option for downloading it from upstream could be also useful.